cmd/earthly/main.go

```diff
@@ -903,6 +903,12 @@ func (app *earthlyApp) before(context *cli.Context) error {
 	app.buildkitdSettings.AdditionalArgs = app.cfg.Global.BuildkitAdditionalArgs
 	app.buildkitdSettings.AdditionalConfig = app.cfg.Global.BuildkitAdditionalConfig
 
+	// ensure the MTU is something allowable in IPv4, cap enforced by type. Zero is autodetect.
+	if app.buildkitdSettings.CniMtu != 0 && app.buildkitdSettings.CniMtu < 68 {
+		return errors.New("Invalid overridden MTU size")
```

Comment:
Nit: lower-case errors

```suggestion
		return errors.New("invalid overridden MTU size")
```

Reply:
I get that idiomatic Go wants lower case errors, but it feels wrong to present sentences that start with lower case to the user.

Reply:
It's actually presented as `Error: invalid overridden MTU size` to the user

Reply:
Still feels wrong to me. But I am willing to follow convention.